src/components/Drawer/AddReportDrawer/AddReportDrawerContent.tsx

```diff
@@ -464,6 +372,45 @@ const StatusValueSelect: React.FC = () => {
   );
 };
 
+const AddDescription: React.FC = () => {
+  const { t } = useTranslation();
+
+  const dispatch = useDispatch();
+  const reportData = useSelector((state: IRootState) => state.report.data);
+
+  const handleInputChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    const newValue = e.target.value;
+    dispatch(
+      updateAddReportData({
+        target: {
+          ...reportData.target,
+          description: newValue,
+        },
+      }),
+    );
```

Comment:
It is better to pass the value directly. No need to create a new parameter.

Reply:
I removed the newValue parameter, and passed e.target.value to description directly.